actions/subscribers.go

```diff
@@ -472,6 +480,13 @@ func BulkRemoveSubscribers(c *gin.Context) {
 		return
 	}
 
+	defer func() {
+		err = res.Body.Close()
```

Comment:
For bulk removal I think we close the body inside the service method, we won't need it here.

Reply:
Yes you are right.